timemachine/datasets/fep_benchmark/hif2a/results_edges_20ns.csv

```diff
@@ -1,93 +1,81 @@
-227.0,231.0,0.74,1.35,0.04,1.15,1.51,N/A,N/A
-227.0,15.0,-0.29,0.26,0.04,0.49,1.31,N/A,N/A
-266.0,156.0,0.8,N/A,N/A,N/A,N/A,N/A,N/A
-266.0,165.0,1.3,1.39,0.12,0.9,1.08,N/A,N/A
-266.0,84.0,2.96,0.66,0.11,1.13,1.08,N/A,N/A
-266.0,254.0,0.98,-0.32,0.11,-0.29,0.71,N/A,N/A
-206.0,57.0,-0.55,-1.97,0.04,-2.15,0.18,N/A,N/A
-206.0,42.0,1.18,-0.86,0.08,-1.05,0.79,N/A,N/A
-206.0,23.0,3.06,0.81,0.07,0.72,0.17,N/A,N/A
-206.0,256.0,-0.55,-0.9,0.18,-0.43,0.79,N/A,N/A
-67.0,165.0,0.94,1.66,0.16,2.16,0.79,N/A,N/A
-67.0,256.0,-0.5,2.54,0.08,2.04,0.79,N/A,N/A
-156.0,167.0,-0.94,-1.47,0.04,-1.55,0.98,N/A,N/A
-156.0,165.0,0.5,0.3,0.11,0.31,0.98,N/A,N/A
-156.0,42.0,0.8,-0.71,0.05,-0.42,0.79,N/A,N/A
-156.0,164.0,2.97,2.69,0.12,2.47,0.84,N/A,N/A
-31.0,61.0,-1.27,-0.1,0.07,-0.06,1.19,N/A,N/A
-31.0,7a,-1.15,0.56,0.09,0.84,0.39,N/A,N/A
-289.0,35.0,3.76,0.14,0.06,0.14,0.06,N/A,N/A
-289.0,224.0,1.25,N/A,N/A,N/A,N/A,N/A,N/A
-61.0,251.0,-0.14,-0.18,0.04,-0.14,1.19,N/A,N/A
-167.0,165.0,1.44,2.17,0.07,1.86,0.9,N/A,N/A
-167.0,155.0,0.17,0.75,0.05,0.9,1.18,N/A,N/A
-167.0,50.0,0.98,1.84,0.07,2.03,0.58,N/A,N/A
-165.0,42.0,0.3,N/A,N/A,N/A,N/A,N/A,N/A
-165.0,252.0,-0.48,-0.18,0.12,-0.53,1.27,N/A,N/A
-165.0,336.0,-0.59,-1.31,0.17,-0.84,1.27,N/A,N/A
-165.0,163.0,-1.77,-1.25,0.13,-0.96,0.77,N/A,N/A
-41.0,7b,2.39,3.14,0.25,4.25,2.51,N/A,N/A
-7b,42.0,-1.55,N/A,N/A,N/A,N/A,N/A,N/A
-7b,231.0,-2.63,-5.23,0.11,-4.12,2.51,N/A,N/A
-54.0,251.0,-0.93,-1.23,0.15,-1.3,0.83,N/A,N/A
-54.0,124.0,-0.81,-1.61,0.09,-1.61,0.83,N/A,N/A
-215.0,252.0,0.89,0.21,0.12,0.47,1.26,N/A,N/A
-215.0,15.0,-0.45,-0.85,0.04,-1.26,1.26,N/A,N/A
-215.0,163.0,-0.41,-0.01,0.05,0.04,0.66,N/A,N/A
-57.0,231.0,0.65,0.43,0.04,0.25,0.18,N/A,N/A
-42.0,164.0,2.17,2.82,0.14,2.9,0.57,N/A,N/A
-35.0,7a,-2.84,2.93,0.06,2.93,0.06,N/A,N/A
-35.0,84.0,-0.42,N/A,N/A,N/A,N/A,N/A,N/A
-252.0,336.0,-0.11,-0.77,0.03,-0.31,2.51,N/A,N/A
-252.0,290.0,-0.85,N/A,N/A,N/A,N/A,N/A,N/A
-252.0,155.0,-0.79,0.01,0.09,-0.43,1.18,N/A,N/A
-252.0,231.0,-0.31,-0.3,0.07,-1.06,2.51,N/A,N/A
-252.0,124.0,-0.41,-1.02,0.13,-0.36,1.09,N/A,N/A
-336.0,251.0,-0.42,0.94,0.27,0.27,1.31,N/A,N/A
-290.0,7a,0.57,2.65,0.12,2.63,0.33,N/A,N/A
-290.0,84.0,2.99,N/A,N/A,N/A,N/A,N/A,N/A
-290.0,23.0,3.5,3.21,0.13,3.2,0.98,N/A,N/A
-290.0,256.0,-0.11,N/A,N/A,N/A,N/A,N/A,N/A
-155.0,84.0,2.93,1.43,0.09,1.19,0.89,N/A,N/A
-164.0,124.0,-3.36,-2.39,0.16,-3.05,1.09,N/A,N/A
-164.0,25.0,-3.52,-2.82,0.13,-2.3,1.09,N/A,N/A
-164.0,163.0,-4.25,N/A,N/A,N/A,N/A,N/A,N/A
-7a,224.0,0.34,-2.18,0.09,-2.15,0.29,N/A,N/A
-7a,23.0,2.93,0.35,0.06,0.57,0.39,N/A,N/A
-224.0,231.0,-0.36,0.06,0.1,0.09,0.29,N/A,N/A
-84.0,15.0,-3.47,-2.7,0.11,-2.49,1.08,N/A,N/A
-254.0,251.0,-0.69,0.5,0.02,0.62,1.21,N/A,N/A
-254.0,50.0,-0.15,1.56,0.08,1.37,0.58,N/A,N/A
-23.0,15.0,-3.98,-3.41,0.06,-3.29,0.98,N/A,N/A
-251.0,15.0,-0.8,-1.61,0.05,-1.68,1.21,N/A,N/A
-251.0,25.0,-0.04,1.04,0.08,0.44,1.31,N/A,N/A
-15.0,25.0,0.76,2.16,0.04,2.12,1.31,N/A,N/A
+Ligand1,Ligand2,Exp.,FEP,FEP Error,CCC,CCC Error

```

Comment:
this file has fewer rows than the lhs because the N/A rows have been removed

Reply:
Hmm, are the rows where `"Exp."` field is a float but `"FEP"` field is a NaN still useful for anything?

Reply:
These are experimental ddGs - which can always be computed from the node dGs directly (present in a different file). So - I don't really think its useful to keep these